ext/reflection/php_reflection.c

```diff
@@ -3496,7 +3496,7 @@ ZEND_METHOD(ReflectionFunctionAbstract, inNamespace)
 
 	zend_string *name = fptr->common.function_name;
 	const char *backslash = zend_memrchr(ZSTR_VAL(name), '\\', ZSTR_LEN(name));
```

Comment:
What is the best/fastest function to detect if one char is present?

Reply:
`zend_memrchr` likely already is the fastest way.